include/inviwo/core/properties/property.h

```diff
@@ -74,7 +74,7 @@ struct PropertyTraits {
      * "T::CLASS_IDENTIFIER". In case it is not found an empty string will be returned. An empty
      * class identifier will be considered an error in various factories.
      */
-    static std::string classIdentifier() { return util::classIdentifier<T>(); }
+    static const std::string& classIdentifier() { return util::classIdentifier<T>(); }
```

Comment:
should be documented in the changelog

Reply:
actually since this is just a static function not virtual or any such things, it does not matter or break any thing.
your specialisation can return whatever...